src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -105,6 +119,8 @@ public function collect(Request $request, Response $response, \Throwable $except
             'response_cookies' => $responseCookies,
             'session_metadata' => $sessionMetadata,
             'session_attributes' => $sessionAttributes,
+            'session_usages' => array_values($this->sessionUsages),
+            'stateless_check' => $requestStateless ?: false,
```

Comment:
`$requestStack->getMasterRequest()->attributes->get('_stateless', false)` here
we don't care if a subrequest is stateless or not, only the master one matters here